test cases/common/223 native prop/meson.build

```diff
@@ -22,4 +22,14 @@ assert(x=='fallback', 'fallback native:false did not work')
 
 
 x = meson.get_external_property('anarray')
-assert(x==['one', 'two'], 'array did not work')
\ No newline at end of file
+assert(x==['one', 'two'], 'array did not work')
+
+assert(meson.has_external_property('anarray'))
+assert(meson.has_external_property('astring'))
+assert(not meson.has_external_property('abool'))
+
+if meson.is_cross_build()
+  assert(meson.has_cross_property('anarray'))
+  assert(meson.has_cross_property('astring'))
+  assert(not meson.has_cross_property('abool'))
+endif
```

Comment:
Could add else case where you check they all return false.

Reply:
Done.